### ISSUE TITLE
H.264 MVC 3D on Mediatek chipsets?

### ISSUE THREAD

**Issue by Skinfaxi** [koying/SPMC#153]
After supporting 3D BR ISOs with the Amlogic chipset in version 16, I am wondering, if there are any plans to bring MVC 3D playback to Mediatek (Sony Bravia >= 2015)?

Currently I am not able to play back BR 3D ISOs on a Sony X85C. The picture is black, but the time ruler is moving (erratically, I might add).

**Comment by koying** [comment]
No known support of MVC from mediatek

**Comment by Skinfaxi** [comment]
@koying 
I discovered something which maybe is of interest for you:

I learned that Sony 3D Camcorders HDR-TD10/20/30 use MVC format.
I found these short sample clips [http://forum.cyberlink.com/forum/posts/list/18211.page](url) Post: 05/08/2011 14:16:48.
The other file from the Panasonic is simple SBS, but the two files from the Sony camcorder are MVC.

To my surprise they play back perfectly in 3D on my Sony X85C (Mediatek chipset!) with the TV's internal Sony Media Player.
Kodi plays them back, too - but only in 2D.
My old Asus O!Play plays them back, but only 2D.
SPMC doesn't play them back (it stays dark, but shows "playing").

A few more observations:
The TV's internal media player app even plays back MVC MKVs, but with errors and pixelated.

Therefore I think the problem lies with the format that MakeMKV uses to create MVC. 

The 3D-camcorder files prove, that MVC works also on Mediatek-chipsets and probably on many other systems.
Is it possible that Makemkv is using a very strange format to insert the MVC stream? It seems so to me.

What does not seem right with the MKV MVC stream is the fact, that it plays back with errors, instead playing back as 2D only, if a unit doesn't understand the format correctly.
